test/php/Nominatim/GeocodeTest.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace Nominatim;
+
+require_once(dirname(dirname(__FILE__)).'/../../build/settings/settings.php');
```

Comment:
We can't hard-code the name of the build directory as it may be called anything. If you need to reference to the build directory, add an environment variable where it can be changed.

The question is though, if you really need to refer to specific settings. I'd expect the unit tests to run independently from the current configuration. In that case it should be enough to include settings/default and add some variables you need for the test harness.